data/rom_85CECB0.s

```diff
@@ -0,0 +1,154 @@
+	.include "asm/macros.inc"
```

Comment:
This file was split from rom_8158B30.s, which was named after its corresponding asm/ file. This should follow a similar naming scheme instead of naming it after the offset of the data it contains.